chromium_src/chrome/browser/net/system_network_context_manager.cc

```diff
@@ -1,8 +1,15 @@
 /* Copyright (c) 2019 The Brave Authors. All rights reserved.
  * This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
- * You can obtain one at http://mozilla.org/MPL/2.0/. */
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+#include "build/branding_buildflags.h"
 
 #include "brave/services/network/public/cpp/system_request_handler.h"
 
+// This is currently necessary in order to enable Certificate Transparency
+// enforcement (brave-browser#22482).
```

Comment:
what about `kCertificateTransparencyAndroid`? is it for another PR?

Reply:
Yes, I was originally thinking of splitting it into two different PRs, but maybe it would be better to enable in both at the same time.

Reply:
> Yes, I was originally thinking of splitting it into two different PRs, but maybe it would be better to enable in both at the same time.

I think if you can make required tests run on Android, then we can do this in the same PR. That's the hard part IMO 😅

Reply:
So this PR already works fine on Android. I've just confirmed it on my phone.

The browsertest (which I copied from upstream) however will not run on Android. It has too many desktop-only dependencies. I looked for other browseertests upstream that run on Android and didn't find any.

It looks like it would be quite hard to come up with a test that works on Android and so I'd like to propose that we rely on the fact that the implementation is common to both desktop (which has a browsertest) and Android. In addition, I'll make it clear in the manual test instructions here that we should verify on both Android and desktop.

Does that sound good to you @goodov ?